src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.Android.cs

```diff
@@ -22,6 +25,17 @@ namespace Uno.Toolkit.UI;
 
 public partial class ExtendedSplashScreen
 {
+	public bool SplashIsEnabled
+	{
+		get
+		{
+			var versionString = Build.VERSION.Release;
+			return (int.TryParse(versionString, out var intVersion) && intVersion >= 12) ?
+						false :
```

Comment:
It does, and at least in the short term this will fix the issue with the native splash not aligning with extended splash. I'm also not convinced we should be using extended splash on Android 12+ since it has a mechanism to keep the native splash visible until the app is ready to render (ie Activate is called)